keychain_haiku.cpp

```diff
@@ -9,10 +9,64 @@
 #include "keychain_p.h"
 
 #include <KeyStore.h>
+
+#include <Application.h>
+#include <AppFileInfo.h>
+#include <File.h>
+
 #include <QDebug>
+#include <QCoreApplication>
+#include <QString>
 
 using namespace QKeychain;
 
+class AutoApp {
+public:
+	AutoApp();
```

Comment:
This commit seems to use tabs, please use four spaces instead.

Reply:
Code looks fine otherwise.

Reply:
Ah yes, that's the default for Haiku…
Do you want it squashed?